fenix/app/src/main/java/org/mozilla/fenix/debugsettings/store/DebugDrawerStore.kt

```diff
@@ -24,5 +24,8 @@ private fun reduce(state: DebugDrawerState, action: DebugDrawerAction): DebugDra
     return when (action) {
         is DebugDrawerAction.DrawerOpened -> state.copy(drawerStatus = DebugDrawerState.DrawerStatus.Open)
         is DebugDrawerAction.DrawerClosed -> state.copy(drawerStatus = DebugDrawerState.DrawerStatus.Closed)
+        is DebugDrawerAction.NavigateTo,
+        DebugDrawerAction.OnBackPressed,
+        -> state // handled by [DebugDrawerNavigationMiddleware]
```

Comment:
nit:
```suggestion
        is DebugDrawerAction.NavigateTo, DebugDrawerAction.OnBackPressed -> state // handled by [DebugDrawerNavigationMiddleware]

```

Reply:
I had to do this because of lint, but I'll try to find something cleaner